test/export/test_export.py

```diff
@@ -1904,17 +1904,6 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         dim0_x = torch.export.Dim("dim0_x", min=3)
         dim1_x = torch.export.Dim("dim1_x", max=8000)
         dynamic_shapes = {"x": (dim0_x, dim1_x)}
-        with self.assertRaisesRegex(
```

Comment:
we no longer do any specializations

Reply:
Nice. Might be good to name the exported program and then do some test to ensure what runtime assert was added?

Reply:
Ah, I should make a note here, this test case does require the complex guards flag to be set, it encounters a guard `Ne(Mod(s0, s0-1), 0)` (special case of reshape) when tracing.

For the cases I've removed, some require this flag, some export fine with just hybrid symints. For those that are fine, the asserts aren't perfect because replacement expressions are set for placeholders, but not checked at runtime. I was planning to leave that as the follow up task (undo replacement, set as runtime assert)